integration_tests/integration/reviewAVisit.cy.ts

```diff
@@ -55,6 +30,31 @@ context('Review a visit', () => {
   })
 
   it('should dismiss notifications when a visit is marked as not needing to be updated or cancelled', () => {
+    const eventsAudit: VisitHistoryDetails['eventsAudit'] = [
```

Comment:
Is there a reason for moving and duplicating these?

Reply:
was so it's clear at the start of the test which eventsAudit items are appearing in each test, instead of manually changing a particular array numbers entry

Reply:
But everything except the type stays the same? Since it could be set in `beforeEach` you would just have to change `eventsAudit[1].type` on each test which is cleaner.